velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -152,4 +152,29 @@ struct UnixTimestampParseWithFormatFunction
   bool invalidFormat_{false};
 };
 
+template <typename T>
+struct LastDayFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE int64_t getYear(const std::tm& time) {
+    return 1900 + time.tm_year;
+  }
+
+  FOLLY_ALWAYS_INLINE int64_t getMonth(const std::tm& time) {
+    return 1 + time.tm_mon;
+  }
+
+  FOLLY_ALWAYS_INLINE bool call(
```

Comment:
use void return type since it never generates null

Reply:
@laithsakka done